File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
var intersection = function(nums1, nums2) {
    let  j = [] 
    for (let i=0;i<nums1.length;i++){
        
        if(nums2.includes(nums1[i])&&!j.includes(nums1[i])){
            j.push(nums1[i])
           
        }
    
    }
    return j
    }

File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.cs
public class Solution {
    public int[] Intersection(int[] nums1, int[] nums2) {
        HashSet<int> str =new HashSet<int>(nums1);
        HashSet<int> result =new HashSet<int>();
        foreach(var num in str){
            if(nums2.Contains(num)){
           result.Add(num);

            }
    
        }
        

     return result.ToArray();    
    }
}